### PR TITLE
Clean up unused dependencies

### PR DESCRIPTION
**Pull request by ccschmitz**
## Summary

Used [depcheck](https://github.com/depcheck/depcheck) to find and clean up some unused dependencies in our frontend code.

## How did you test this change?

Local click test seemed to work as expected.

## Are there any deployment considerations?

I think if click test and CI checks pass we are probably in good shape.

### CODE REVIEW

File: frontend/package.json [hunk 1]
```diff
@@ -12,19 +12,16 @@
 		"@highlight-run/client": "workspace:*",
 		"@highlight-run/react": "workspace:*",
 		"@highlight-run/react-mentions": "4.4.1",
-		"@highlight-run/react-select": "^5.2.2",
```
Comment (ccschmitz) [inline frontend/package.json:15]: We also have the main `react-select` package installed and are importing from that everywhere.

**Review by Vadman97 — APPROVED** [review]
render.com preview looks good throughout our products

File: frontend/package.json (hunk 1 above)
Comment (ccschmitz) [reply frontend/package.json:15]: Yeah, I was thinking we of doing that as a follow up. Wanted to make sure we weren't trying to install it in CI before cleaning it up.

Comment (ccschmitz) [reply frontend/package.json:15]: I did end up archiving this: https://github.com/highlight/react-select

Comment (Vadman97) [reply frontend/package.json:15]: thank you!

**Pull request merged**